ui/src/app/store/pod/types.ts

```diff
@@ -8,6 +8,7 @@ export enum PodType {
   VIRSH = "virsh",
 }
 
+// Remove when https://bugs.launchpad.net/maas/+bug/1928442 lands
```

Comment:
Should we just remove all this now? We don't want to use these types.

Reply:
I guess it feels a little weird to be ahead of the api, but yeah we're not gonna use these values again so they're not useful to keep.